src/main/git-elegant

```diff
@@ -85,21 +85,9 @@ __batch() {
     fi
 }
 
-feature() {
-    _validate "$1" "Feature name"
-    git-elegant pull $MASTER
-    git checkout -b "$1"
-}
-
-clear-local() {
-    git-elegant pull $MASTER
-    local cmd="git branch -lvv | grep gone | awk {'print \$1'}"
-    __loop "git branch -d" $(eval "$cmd") || \
-        (
-            __gm "There are unmerged branches:" && \
-            __loop "__gm -" $(eval "$cmd") && \
-            __batch "Do you want to delete all unmerged branches?" "Delete this?" "git branch -D" $(eval "$cmd")
-        )
+rebase() {
```

Comment:
@extsoft `rebase` is also moved to separate executable, so we don't need it here.

Reply:
Done - https://github.com/extsoft/elegant-git/pull/78/commits/792a9972c302e8632676c15375d83bb9fe492403